README.md

```diff
@@ -142,7 +142,13 @@ To install HIE, you need stack version >= 1.7.1.
 
 HIE builds from source code, so there's a couple of extra steps.
 
-#### Linux pre-requirements
+#### Common pre-requirements
+
+* `stack` must be in your PATH
```

Comment:
Maybe clarify that the `stack` command must be fully functioning? 
E.g. on ubuntu, I installed stack and `stack setup` failed out of the box?
But maybe we can expect that everybody understands that in your path means, it must be working?

Reply:
Hm. IIRC, stack will complain if `stack setup` wasn't ran before and provide a suggestion to run `stack setup`.
I can also add a small clarification like "If you run `git --version` and see an output instead of 'command not found' message, that means that git is in your PATH".

Reply:
OTOH, all that time before wording was the same and there were no issues regarding git/stack/whatever not being in PATH or stack setup. But i guess i'll add clarification anyway.

Reply:
You dont have to, you are probably right.